app/models/miq_alert_status_action.rb

```diff
@@ -0,0 +1,30 @@
+class MiqAlertStatusAction < ApplicationRecord
```

Comment:
@chrisarcand that might be exactly what I need, I need to allow deletion of status actions only when the owning status is destroyed. I could not find a way to use readonly except as a callback:
```ruby
def readonly?
  !new_record?
end
```

I have a feeling you meant something else?